gluonnlp/base.py

```diff
@@ -27,4 +27,4 @@
 if sys.version_info[0] == 3:
     _str_types = (str, )
 else:
-    _str_types = (str, unicode)
+    _str_types = (str, unicode)  # noqa: F821
```

Comment:
Instead of the __noqa__ approach, you might substitute in the changes made in #215 and then I can close #215.